src/common/config_validator_adapter.tsx

```diff
@@ -1,19 +1,19 @@
 import { ValidatorFactory } from '../schemas/validators';
 import { EngagementFormConfig } from '../schemas/engagement_config';
 
-export const getValidatorsFromFormOptions = (
-  formOptions: EngagementFormConfig = {}
+export const getValidatorsFromengagementFormConfig = (
```

Comment:
Should the e in engagement be an E?  IE - proper camel case - will that cause accident/confusion?